### ISSUE TITLE
Error running on OSX: Cannot read property 'exists' of undefined

### ISSUE THREAD

**Issue by jfrumar** [Benvie/Node.js-Ultra-REPL#33]
I tried both `npm install ultra-repl` and `npm install https://github.com/Benvie/Node.js-Ultra-REPL.git`.

When I run node_modules/ultra-repl/bin/ultra-repl.js I receive this:

```
if (NativeModule.exists(request)) {
                    ^
TypeError: Cannot read property 'exists' of undefined
    at Function._resolveFilename (/Users/jules/Documents/workspace/groupahead-build-system-ver2/node_modules/ultra-repl/lib/ScopedModule.js:81:21)
    at Function._load (/Users/jules/Documents/workspace/groupahead-build-system-ver2/node_modules/ultra-repl/lib/ScopedModule.js:102:33)
    at Object.<anonymous> (/Users/jules/Documents/workspace/groupahead-build-system-ver2/node_modules/ultra-repl/index.js:3:30)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
```

**Comment by boneskull** [comment]
Yeah, this looked cool, but it hasn't been touched in 3 years and might not even work on non-Win32 OS's.  It's using some weird stuff (what's `.imports.json`?).  I forked it to try to fix the issue but I quickly went down a rabbit hole.

**Comment by Pysis868** [comment]
Also happening here.

Instead of `npm install ultra-repl` I used `npm install --global ultra-repl` so I could get the command from my PATH.

Mac OS X 10.11.1
Node v0.12.7
NPM 2.12.1

Pretty sure it's a project issue but hey, just adding this info to help.

**Comment by cjwelborn** [comment]
Same on linux.

App | Version
--- | ---
| Kubuntu | 15.10 |
| Node | 0.12.9 |
| Npm | 2.14.9 |

**Comment by kedare** [comment]
Same on Ubuntu 14.04 (node 5.3.0) and Windows 10 (node v5.6.0)

**Comment by Lucas-C** [comment]
Same here using Cywgin, Node v0.12.2 and npm v3.8.7

**Comment by ottonascarella** [comment]
same in macos 10.12.3 with node 7.6.0

**Comment by cjwelborn** [comment]
@ottonascarella, @Lucas-C, and everyone else, it looks like the author of this tool is not responding to issues. I've found a really good Node REPL called "[Mancy](http://mancy-re.pl/)". It's electron-based and has support for several third-party libraries (like React). Check it out: http://mancy-re.pl/
